README.md

```diff
@@ -4,6 +4,12 @@ React webapp that can read zipped manga files from url. Built for KaratsubaScans
 
 ## Running for development
 
+Docker:
+
+```
+docker-compose up
```

Comment:
when giving code examples for shell commands, i like to place a prompt character to indicate that it's a command
```
$ docker-compose up
```